### ISSUE TITLE
Question: Call finetuned models for metric handler

### ISSUE THREAD

**Issue by Seung-B** [locuslab/open-unlearning#131]
Hello!

I want to add the metric handler with my own metric for experiments.
My metric is designed to compare the outputs of the finetuned model and the unlearned model.
So need to use tofu-finetuned model.
How can I call fine-tuned models in metric handler?
